server/sonar-web/src/main/js/components/issue/issue-view.js

```diff
@@ -253,7 +253,7 @@ export default Marionette.ItemView.extend({
   serializeData () {
     const issueKey = encodeURIComponent(this.model.get('key'));
     return _.extend(Marionette.ItemView.prototype.serializeData.apply(this, arguments), {
-      permalink: '/issues/search#issues=' + issueKey,
+      permalink: window.baseUrl + '/issues/search#issues=' + issueKey,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)